src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="form"
+      data-qa="big"
+    >
+      <label class="search__bar search__element">
+        <div class="search__image"></div>
```

Comment:
better to use img tag instead of the div

Reply:
Hi, thank`s for your check! The first item in checklist is: "Icon implemented using background-image CSS property". If I use tag image I can`t apply background-image. Please explain how to so that thing)